# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1497393378 14400
#      Tue Jun 13 18:36:18 2017 -0400
# Node ID a990b6d740c49e7e1860d94ba165265c14966115
# Parent  eb3c7abad98f040fe09a1dfe8f825c4c0ad97b7d
* gzip.cc: Fix syntax error in test.

diff --git a/libinterp/dldfcn/gzip.cc b/libinterp/dldfcn/gzip.cc
--- a/libinterp/dldfcn/gzip.cc
+++ b/libinterp/dldfcn/gzip.cc
@@ -766,17 +766,17 @@ The optional output @var{filelist} is a 
 %!
 %!  md5_z = hash ("md5", fileread (z_file));
 %!  unlink_or_error (z_file);
 %!  uz_z_filelist = z.unzip (z_z_file);
 %!  assert (uz_z_filelist, {z_file}) # bug #48598
 %!  assert (exist (z_z_file), 2) # bug #48597
 %!  assert (hash ("md5", fileread (z_file)), md5_z)
 %!endfunction
-%!xtest run_test_function (@test_z_z) <48597>
+%!xtest <48597> run_test_function (@test_z_z)
 
 %!function test_xzip_dir (test_dir, z) # bug #43431
 %!  fpaths = fullfile (test_dir, {"test1", "test2", "test3"});
 %!  md5s = cell (1, 3);
 %!  for idx = 1:numel(fpaths)
 %!    create_file (fpaths{idx}, rand (100, 1));
 %!    md5s(idx) = hash ("md5", fileread (fpaths{idx}));
 %!  endfor
